gtsfm/evaluation/visualize_benchmark_comparison.py

```diff
@@ -64,7 +64,7 @@ def colorscale_from_list(colorlist: List[str]) -> List[str]:
     return colorscale
 
 
-def plot_colored_table(row_labels: List[str], col_labels: List[str], tab_data: np.ndarray) -> str:
+def plot_colored_table(new_values: np.ndarray, old_values: np.ndarray, row_labels: List[str], col_labels: List[str], tab_data: np.ndarray) -> str:
```

Comment:
looks good @nehau9.

This line is a bit long (147 chars) -- can you reformat the file with 120-char max length?
`black gtsfm/evaluation/visualize_benchmark_comparison.py -l 120`
after `pip install black`
Thanks!

Reply:
could you add info about `new_values` and `old_values` to the docstring `Args` also?

Reply:
i made these updates, should be good to go